composer/models/tasks/classification.py

```diff
@@ -45,23 +51,34 @@ class ComposerClassifier(ComposerModel):
 
     num_classes: Optional[int] = None
 
-    def __init__(self, module: torch.nn.Module) -> None:
+    def __init__(self, module: torch.nn.Module, loss_name: str = "soft_cross_entropy") -> None:
         super().__init__()
         self.train_acc = Accuracy()
         self.val_acc = Accuracy()
         self.val_loss = CrossEntropy()
         self.module = module
+        if loss_name not in loss_registry.keys():
+            raise ValueError(f"Unrecognized loss function: {loss_name}. Please ensure the "
+                             "specified loss function is present in composer.loss.loss.py")
+        self._loss_fxn = loss_registry[loss_name]
 
         if hasattr(self.module, "num_classes"):
             self.num_classes = getattr(self.module, "num_classes")
 
+        if loss_name == 'binary_cross_entropy_with_logits':
+            log.warning("UserWarning: Using `binary_cross_entropy_loss_with_logits` "
+                        "without using `initializers.linear_log_constant_bias` can degrade "
+                        "performance. "
+                        "Please ensure you are using `initializers. "
+                        "linear_log_constant_bias`.")
+
```

Comment:
I think Ravi likes to use textwrap.dedent(""" """) for these long strings, but probably not necessary?